ui/Decoder.elm

```diff
@@ -13,7 +13,7 @@ buildRecordDecoder =
         |> required "build" buildDecoder
         |> required "download" downloadDecoder
         |> required "source" sourceDecoder
-        |> required "systemaddons" (nullable (list systemAddonDecoder))
+        |> required "systemaddons" (maybe (list systemAddonDecoder))
```

Comment:
But now I think more about it, don't we have an overall schema issue here? I mean, does it make sense to have a `SystemAddon` entry with no `builtin` and/or `updated` fields? Sounds like an inconsistent model where we make impossible states possible...